commands/command_env.go

```diff
@@ -16,7 +16,7 @@ var (
 func envCommand(cmd *cobra.Command, args []string) {
 	lfs.ShowConfigWarnings = true
 	config := lfs.Config
-	endpoint := config.Endpoint()
+	endpoint := config.Endpoint("download")
```

Comment:
Yeah, I was thinking of doing that but there's lots of other places (unrelated to this change) it should apply too; it was already leaking into more areas than I liked so I didn't do it. I can do though.